README.md

```diff
@@ -26,6 +26,14 @@ Player with the most points wins.
 
   * If the cue ball is pocketed, any points accumulated for **this shot** are awarded to the opposing player.
     The turn ends, and the opposing player has ball in hand.
+  * If the cue ball leaves the table, any points accumulated for **this shot** are awarded to the opposing player.
```

Comment:
We should define a formal description of a scratch. Every result of a scratch (cue pocketed, cue off table, etc) has the same consequence right?